portal/exercise_diet/templates/exercise_diet/exercise.html

```diff
@@ -13,23 +13,23 @@
     {{super()}}
     {% for id, content in modals.items() %}
         <!-- Modal -->
-        <div class="modal fade" id={{ id }} aria-hidden="true">
+        <div class="ED__modal modal fade" id={{ id }} aria-hidden="true">
             <div class="modal-dialog">
                 <div class="modal-content">
                     <div class="modal-body">
                         <button type="button" class="btn btn-secondary-top" data-dismiss="modal"><b>close</b><span>X</span></button>
-                        {{ content[2]|safe }}
+                        <div class="content">{{ content[2]|safe }}</div>
                     </div>
                     {% if content[1] > 0 %}
                     <div class="modal-nav">
                         {% if (loop.previtem is defined) and (loop.previtem[1][1] > 0) %}
                         <span class="left-button">
-                            <a href="javascript:show_exercise('{{ loop.previtem[0] }}','{{ id }}');">&larr;&nbsp;{{ loop.previtem[1][0]|safe }}</a>
+                            <a href="javascript:show_exercise('{{ loop.previtem[0] }}','{{ id }}');"><span class="figure">&larr;</span><span>&nbsp;{{ loop.previtem[1][0]|safe }}</span></a>
```

Comment:
Probably out of scope for this PR, but would you be able to remove this inline JS? 

I think you could remove it by assigning `loop.previtem[0]` and `id` as HTML attributes, then (in `exercise_diet.js`) loop over them and call `show_exercise()` on each HTML element, passing the attributes

Reply:
:medal_sports: